<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtClass</class>
 <widget class="QMainWindow" name="qtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>791</width>
      <height>481</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="tabBarAutoHide">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>首页</string>
     </attribute>
     <widget class="QPushButton" name="changebtn_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>120</y>
        <width>188</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>功能二</string>
      </property>
     </widget>
     <widget class="QPushButton" name="changebtn_3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>180</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>功能三</string>
      </property>
     </widget>
     <widget class="QPushButton" name="changebtn_1">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>60</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>功能一</string>
      </property>
     </widget>
     <widget class="QPushButton" name="changebtn_4">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>230</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>功能四</string>
      </property>
     </widget>
     <widget class="QPushButton" name="changebtn_5">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>280</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>功能五</string>
      </property>
     </widget>
     <widget class="QPushButton" name="changebtn_6">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>330</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>功能六</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>功能一</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>360</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="returnbtn_1">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>350</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="func1_line1">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>160</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="func1_re">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>220</y>
        <width>491</width>
        <height>87</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="func1_line2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>160</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>起始地址：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>130</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>线圈个数</string>
      </property>
     </widget>
     <widget class="QPushButton" name="func1_send">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>160</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>功能二</string>
     </attribute>
     <widget class="QPushButton" name="returnbtn_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>330</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QPushButton" name="func2_send">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>200</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="func2_line1">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="func2_line2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>200</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>170</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>起始地址</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>170</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>离散值个数</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="func2_re">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>280</y>
        <width>321</width>
        <height>87</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>功能三</string>
     </attribute>
     <widget class="QPushButton" name="returnbtn_3">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>260</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QPushButton" name="func3_send">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>150</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="func3_line1">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="func3_line2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>160</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="func3_re">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>220</y>
        <width>104</width>
        <height>87</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>功能四</string>
     </attribute>
     <widget class="QPushButton" name="returnbtn_4">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>280</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="func4_line1">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="func4_line4">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>150</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="func4_send">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>170</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="func4_re">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>240</y>
        <width>104</width>
        <height>87</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>功能五</string>
     </attribute>
     <widget class="QPushButton" name="returnbtn_5">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>240</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="func5_line1">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="func5_line2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>140</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="func5_send">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>140</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="func5_re">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>230</y>
        <width>104</width>
        <height>87</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>功能六</string>
     </attribute>
     <widget class="QPushButton" name="returnbtn_6">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>260</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="func6_line1">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="func6_line2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>130</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="func6_send">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>130</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="func6_re">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>210</y>
        <width>104</width>
        <height>87</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt.qrc"/>
 </resources>
 <connections/>
</ui>
